cmd/snapctl/main.go

```diff
@@ -38,6 +39,19 @@ var clientConfig = client.Config{
 }
 
 func main() {
+	// check for internal commands
+	if len(os.Args) > 2 && os.Args[1] == "internal" {
+		switch os.Args[2] {
+		case "configure-core":
+			if err := corecfg.Run(); err != nil {
+				fmt.Fprintf(os.Stderr, "cannot run core-configure: %s\n", err)
```

Comment:
This should probably be "core configuration error: %v", as the error is actually happening while running it. The message makes it feel like it didn't run.